Makefile

```diff
@@ -81,20 +84,9 @@ deploy: ## Deploy TriggerMesh stack to default Kubernetes cluster using ko
 undeploy: ## Remove TriggerMesh stack from default Kubernetes cluster using ko
 	$(KO) delete -f $(BASE_DIR)/config
 
-release: ## Build release binaries
-	@set -e ; \
-	for bin in $(COMMANDS) ; do \
-		for platform in $(TARGETS); do \
```

Comment:
`$(TARGETS)` is possibly unused as the result of this change?

Reply:
fixed